themes/default/content/docs/guides/saml/auth0.md

```diff
@@ -13,7 +13,7 @@ aliases:
 ---
 
 This guide walks you through configuring your Auth0 Authentication Platform as a SAML SSO identity provider
-(IDP) for the Pulumi Service.
+(IDP) for the Pulumi Cloud.
```

Comment:
```suggestion
(IDP) for Pulumi Cloud.
```

Reply:
@jkodroff lets leave this as it was before this pr opened, we can come back later if we establish a standard around the.